iib/workers/tasks/opm_operations.py

```diff
@@ -918,3 +918,17 @@ def verify_operator_exists(
             is_package_in_index = True
             log.info("operator package %s found in index_db %s", operator_package, index_db_path)
     return is_package_in_index, index_db_path
+
+
+def opm_validate(config_dir: str) -> None:
+    """
+    Validate the declarative config JSON files in a given directory.
```

Comment:
nitpick: I think that `opm validate` does not do difference between YAML or JSON. So we can remove the word `JSON` from here. And below (docstring and log message)

Reply:
I've removed the _JSON_ word from docstrings